NotesApp/Reusable/CustomAddButton.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomAddButton.swift
+//  NotesApp
+//
+//  Created by Luka Gujejiani on 31.10.24.
+//
+
+import UIKit
+
+final class RoundedIconButton: UIButton {
+    // MARK: - Initializers
+    init(iconName: String, pointSize: CGFloat = 12, backgroundColor: UIColor = .darkGray, tintColor: UIColor = .white, cornerRadius: CGFloat = 25, size: CGFloat = 70) {
```

Comment:
Fix indentation of init. Avoid hardcoded values for sizing. Create Constants seperate file or extension inside this file to have sizing there. Also you can have function for configuring ui and then call it in init